src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -384,6 +385,20 @@ public static function copyFile(IOInterface $io, $projectRoot, $source, $target,
     }
 
     /**
+     * Remove line from content.
+     *
+     * @param string $entry Entry to remove.
+     * @param string $content String to remove entry from.
+     * @return string
+     */
+    public static function removeLine($entry, $content)
+    {
+        $entry = preg_quote($entry, '/');
+
+        return preg_replace("/$entry\\W*/", '', $content);
```

Comment:
Let's redo this method to:

```php
return preg_replace(sprintf('/^%s$/m', preg_quote($entry, '/'), '', $content);
```

Using the `m` flag to the regex means that it will only match whole lines, and adding the `^` and `$` anchors ensures a partial match doesn't trigger removal of the line.